File: SPOJ/C++/PRIMO/primo.py
def ePrimo(numero):
    if(numero == 0 or numero == 1): 
        return "nao"
    for i in range(2, numero):
        if(numero % i == 0):
            return "nao"
    return "sim"

numero = int(input())
print(ePrimo(numero))

File: SPOJ/C++/PRIMO/primo.cpp
#include <iostream>
#include<cmath>
#include <string>
using namespace std;

string testaNum(long int);
int main()
{
    long int n;
    string resp;
    scanf("%ld", &n);
    n = fabs(n);
    resp = testaNum(n);
    cout<<"\n"<<resp;
    return 0;
}

string testaNum(long int num){
   long int i;
   if(num == 0 || num == 1) return "nao";
   for(i = 2; i < num; i++){
        if(num % i == 0) return "nao";
   }
   return "sim";
}
